src/components/FlawHistory.vue

```diff
@@ -62,4 +82,51 @@ function isDateField(field: string) {
       </label>
     </div>
   </LabelCollapsible>
+  <div v-if="history" class="pagination-controls gap-1 my-2">
```

Comment:
Since we have the same condition on the two nodes (LabelCollapsible and this div), what do you think about using a `template` to wrap them?

```vue
<template>
  <template v-if="history?.length">
    <LabelCollapsible>
    </LabelCollapsible>
    <div>
    </div>
  </template>
</template>
```